src/Button/Button.module.css

```
@@ -50,3 +49,11 @@
 .Button--secondary:hover {
   box-shadow: inset 0 0 0 2px var(--brand-color-neutral-emphasisPlus);
 }
+
+.Button--size-medium {
+  padding: 1rem 1.5rem;
+}
+
+.Button--size-large {
+  padding: 1rem 2rem;
```

Same as previous comment:

can we update these to use the css variables please? 

```suggestion
  padding: var(--base-size-16) var(--base-size-32);
```